arch/sim/src/sim/sim_alsa.c

```diff
@@ -116,6 +177,32 @@ static const struct audio_ops_s g_sim_audio_ops =
   .release       = sim_audio_release,
 };
 
+static const struct sim_codec_ops_s g_mp3_dec =
+{
+  AUDIO_FMT_MP3,
+  AUDCODEC_DEC,
+  sim_audio_mp3_init,
+  sim_audio_mp3_samples,
+  sim_audio_mp3_decode,
+  sim_audio_mp3_uninit
+};
+
+static const struct sim_codec_ops_s g_pcm_codec =
+{
+  AUDIO_FMT_PCM,
+  AUDCODEC_DEC | AUDCODEC_ENC,
+  sim_audio_pcm_init,
+  NULL,
+  sim_audio_pcm_process,
+  sim_audio_pcm_uninit,
+};
+
+static const struct sim_codec_ops_s g_codec_s[] =
```

Comment:
can we
```suggestion
static const struct sim_codec_ops_s *g_codec_s[] =
```
and use reference to `g_pcm_codec` and `g_mp3_dec` instead of copy?

Reply:
Hi pkarashchenko:
there is no copy.  In func `sim_audio_config_ops`,  use reference of g_pcm_codec or g_mp3_dec.